SingularityUI/app/actions/api/base.es6

```diff
@@ -70,14 +48,46 @@ export function buildApiAction(actionName, opts = {}) {
     return { type: STARTED };
   }
 
-  function error(error) {
-    return { type: ERROR, error };
+  function error(ex) {
+    return { type: ERROR, ex };
```

Comment:
This was causing a linting error.
But it's still probably better to have a clear name.